code/modules/reagents/reagents/other.dm

```diff
@@ -658,3 +658,19 @@
 /datum/reagent/sterilizine/on_mob_life(mob/living/L, metabolism)
 	L.adjustToxLoss(effect_str)
 	return ..()
+
+/*
+/datum/reagent/virilyth
+	name = "Virilyth"
+	description = "A strange substance with high regenerative properties."
+	color = "#488C54"
+	taste_description = "slimy lemon"
+	can_synth = FALSE
```

Comment:
Gonna assume those is wip

Reply:
this is still commented out for some reason?

Reply:
Will either remove it for a later PR or add it with a way to extract green blood from the suit and mix it with certain common reagents to duplicate them.

Depends on how the meds economy goes during the testmerge.